core/keyboard_nav/key_map.js

```diff
@@ -124,13 +129,37 @@ Blockly.user.keyMap.serializeKeyEvent = function(e) {
   return key;
 };
 
+/**
+ * Create the serialized key code that will be used in the key map.
+ * @param {!number} keyCode Number code representing the key.
+ * @param {!Array<string>} modifiers List of modifiers to be used with the key.
+ *     All valid modifiers can be found in the Blockly.user.keyMap.modifierKeys.
+ * @return {string} The serialized key code for the given modifiers and key.
+ */
```

Comment:
public on purpose?

Reply:
Yeah, it is so developers can easily create a keycode for something like control K and then set it for an action.

Reply:
Sounds good.